js/landingsside/dine-oppgaver/DineOppgaverContainer.js

```diff
@@ -130,6 +131,22 @@ NySykepengesoknad.propTypes = {
     soknader: PropTypes.arrayOf(soknadPt),
 };
 
+export const ReisetilskuddSoknad = ({ reisetilskudd }) => {
+    const alleSoknader = [...reisetilskudd];
+    const url = getReisetilskuddSoknaderUrl();
+    console.log('alleSoknader reisetilskudd', alleSoknader); // eslint-disable-line
+    const tekst = alleSoknader.length === 1
```

Comment:
denne vil jo vise for absolutt alle søknader? Jeg ser for meg forskjellig tekst på ÅPEN og SENDBAR, men det bør Bendik men noe om